README.md

```diff
@@ -23,10 +23,21 @@ For more information, see:
 
 This project exists thanks to all the people who contribute and who have contributed in the past, whether as part of the long history of thousands of contributions to WordPress from many different people, or as contributions to ClassicPress itself.
 
-You can see a list of WordPress contributors by going to the [WordPress releases page](https://wordpress.org/news/category/releases/) and looking at the credits for each release.  All changes in WordPress 4.9.8 and earlier are part of ClassicPress, along with selected changes from later releases.
+You can help Fix bugs by submitting Pull Requests. Most of the bugs are shared via the [issues page](https://github.com/ClassicPress/ClassicPress/issues). Here are some pointers on how to:
 
-We are working on several ways to recognize people for their contributions to ClassicPress.  In the meantime, you can view a list of code contributions [here on GitHub](https://github.com/ClassicPress/ClassicPress/compare/LAST_WP_COMMIT...develop).
+- Fix Bugs: Generally when evaluating bugfix PRs, we look for these things (ideally all of them)
+
+  - The change impacts existing ClassicPress users. (Otherwise, there are literally thousands of things we could look at, but we need to prioritize our development time.)
+  - The change is not going to break any other use cases.
+  - The change has automated tests.
+  - We understand the change very well or can ask questions of someone who understands it very well.
+
+* [Backporting Changesets from WordPress](https://classicpress.slack.com/files/UCFQJPU4Q/FFV8NM0TZ/How_to_backport_a_WordPress_changeset)
```

Comment:
This link only works for people who already have a Slack account (try it in an incognito window).  We need to include this content in CONTRIBUTING.md too.

Also, this explains *how* to backport a WordPress changeset, but it doesn't explain *why* we want to do this, or how to choose which changesets to port.

As a starting point:  The same 4 criteria should apply to backports as to bugfixes.

Finally this new document about backporting needs a good amount of grammatical cleanup before it can be included.